smac/tae/execute_func.py

```diff
@@ -160,7 +165,7 @@ def run(self, config, instance=None,
             runtime = float(obj.wall_clock_time)
         else:
             start_time = time.time()
-            result = self.ta(config, **obj_kwargs)
+            result = self._call_ta_raw(config, **obj_kwargs)
```

Comment:
Could you please make the two ways to call this more similar by also allowing additional run information if the pynisher is not used?